buildSrc/linux.gradle

```diff
@@ -232,7 +222,7 @@ FileTree ft_gtk = fileTree("${project(":graphics").projectDir}/src/main/native-g
 LINUX.glass.glass = [:]
 LINUX.glass.glass.nativeSource = ft_gtk_launcher.getFiles()
 LINUX.glass.glass.compiler = compiler
-LINUX.glass.glass.ccFlags = [ccFlags, gtk2CCFlags,  "-Werror"].flatten()
+LINUX.glass.glass.ccFlags = [ccFlags, gtk2CCFlags, gtk3CCFlags, "-Werror"].flatten()
```

Comment:
while this compiles, it might be confusing as it will pull in both the gtk2 and gtk3 headers and there is overlap.

Reply:
Hi Johan,
Thanks for verifying the change, I also verified build by removing the gtk flags.
I shall remove these flags and update the PR.